2021q4/doceng.adoc

```diff
@@ -0,0 +1,48 @@
+////
+Quarter:	4th quarter of 2021
+Prepared by:	dbaio
+Reviewed by:	carlavilla, bcr
+Last edit:	$Date: 2022-01-05 20:07:39 -0300 (Wed, 05 Jan 2022) $
+Version:	$Id: doceng-2021-4th-quarter-status-report.adoc 208 2022-01-05 23:07:39Z dbaio $
+////
+
+=== Documentation Engineering Team
+
+Link: link:https://www.freebsd.org/docproj/[FreeBSD Documentation Project] +
+Link: link:https://docs.freebsd.org/en/books/fdp-primer/[FreeBSD Documentation Project Primer for New Contributors] +
+Link: link:https://www.freebsd.org/administration/#t-doceng[Documentation Engineering Team]
+
+Contact: FreeBSD Doceng Team <doceng@FreeBSD.org>
+
+The doceng@ team is a body to handle some of the meta-project issues associated with the FreeBSD Documentation Project; for more information, see link:https://www.freebsd.org/internal/doceng/[FreeBSD Doceng Team Charter].
+
+No new documentation commit bit was granted during the last quarter, and only one commit bit was safe kept.
+
+Several tasks were made related to the doc tree during the last quarter:
+
+* A COPYRIGHT file was added in the root directory of the doc repository.
++
+The license was also updated to reflect the current toolchains the project is using now.
```

Comment:
My understanding is that it should be "tools" (plural) or "toolchain" (singular, collective), but better have a native speaker check me. (Or it might be several separate sets of tools, in which case "toolchains" would be correct, but I don't think that's the case.)